ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs

```diff
@@ -10,6 +10,6 @@ public interface IUserRepository
 
         Task<IEnumerable<UserDTO>> GetUsers();
 
-        Task<IEnumerable<UserDTO>> GetUsersOfDepartment(int departmentId);
+        Task<IEnumerable<UserForCalendarDTO>> GetUsersOfDepartment(int departmentId);
```

Comment:
User of departments aren't related to calendar, they are still just users of the department. It doesn't make sense.

Reply:
Renamed